.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -60,6 +60,7 @@ jobs:
       user_docs_test: ${{ env.IS_PULL_REQUEST == 'false' ||
         steps.changed-files.outputs.user_docs_any_changed ||
         steps.changed-files.outputs.dependencies_any_changed }}
+      ci_file_changed: ${{ env.IS_PULL_REQUEST == 'false' || steps.changed-files.outputs.ci_any_changed }}
```

Comment:
the token below in the checkout tfhe-rs step would not be available in a pull_request_target event from outside, correct ?

Reply:
Yes it's available, but it's a special token that has only read permission on the repo.
The way it will work is: it'll pull this workflow version from base branch. Then the `should-run` will be executed in a safe way. Finally, if the triggering actor is not member of Zama, the workflow will fail with no possibility to leak this very secret, or any other in fact.

Reply:
isn't the github token enough for a checkout ?

Reply:
Nope because GitHub token only works on public repositories.